source/users/custom-metadata.txt

```diff
@@ -107,6 +107,67 @@ collection. Depending on your use case, you may:
   standard CRUD operations from a Function, an Atlas Device SDK, a MongoDB
   driver, or MongoDB Compass.
 
+Modify User Permissions in Custom User Data Documents
```

Comment:
I think a more apt title might be along the lines of "Best Custom User Data Practices for Automatic Refreshing Permissions"

Reason being, I feel like the current title gives the idea that custom user data documents alone can set permissions, when in reality its all dependent on the roles the developer has set up.

Reply:
Simplified to Permissions and Custom User Data. I can workshop the title with copy reviewer, thanks!